src/main/scala/com/exasol/cloudetl/parquet/ParquetValuesConverter.scala

```diff
@@ -0,0 +1,24 @@
+package com.exasol.cloudetl.parquet
+
+import com.exasol.cloudetl.source.ValueConverter
+import com.exasol.common.data.Row
+import com.exasol.common.json.JsonMapper
+
+import org.apache.parquet.schema.MessageType
+import org.apache.parquet.schema.Type.Repetition
+
+final case class ParquetValueConverter(schema: MessageType) extends ValueConverter {
```

Comment:
Please add a small doc explaining which kind of conversion it is

Reply:
Fixed.